@here/harp-map-theme/resources/berlin_tilezen_day_reduced.json

```diff
@@ -609,26 +589,6 @@
                             2
                         ]
                     },
-                    "secondaryWidth": {
```

Comment:
Same as above.

Reply:
Properties of solid-line technique, not dashed-line.